dockerfiles/stopall.sh

```diff
@@ -18,8 +18,29 @@
 ##############################################################################
 
 # Utility script for stopping a local cluster started by startall.sh
+wait=0
+
+while [ "$#" -gt 0 ]; do
+  arg=$1
+  case "${arg}" in
+      -w) wait=1
+         ;;
+      *) echo "Usage: stoptall.sh [options]"
+         echo "    -w  Wait for all pods to terminate"
+         exit
+         ;;
+    esac
+  shift
+done
 
 helm uninstall mycluster
 helm uninstall localfile
 kubectl delete jobs --all 
 kubectl delete networkpolicy --all 
+if [[ $wait == 1 ]] ; then
+  sleep 2
+  while (kubectl get pods | grep -q ^NAME) ; do
+    echo Waiting...
```

Comment:
trivial: as it's our internal utility script may not matter, but there could be other pods in same namespace that this picks up, that are not to with hpcc.
Perhaps log matches it's still waiting for?

Reply:
Or don't use -w in cases where it's not useful...